tests/integration/targets/aci_domain_to_encap_pool/tasks/main.yml

```diff
@@ -319,6 +330,7 @@
     - phys_vlan_no_alloc.msg is match("ACI requires the 'pool_allocation_mode' for 'pool_type' of 'vlan' and 'vsan' when 'pool' is provided")
     - phys_missing_pool_type.msg is match("missing required arguments{{':'}} pool_type")
     - phys_missing_domain_type.msg is match("missing required arguments{{':'}} domain_type")
+  when: query_cloud.current == []
```

Comment:
As you have put the conditions on all the asserts. Let's do a block for it as there is no reason to run the rest on non-cloud if you don't assert anything on it.